src/libraries/System.Net.Http.Json/src/System/Net/Http/Json/HttpClientJsonExtensions.Delete.cs

```diff
@@ -89,6 +149,16 @@ public static partial class HttpClientJsonExtensions
             return DeleteFromJsonAsyncCore(taskResponse, type, context, cancellationToken);
         }
 
+        /// <summary>
+        /// Sends a DELETE request to the specified Uri and returns the value that results from deserializing the response body as JSON in an asynchronous operation.
+        /// </summary>
+        /// <typeparam name="TValue">The target type to deserialize to.</typeparam>
+        /// <param name="client">The client used to send the request.</param>
+        /// <param name="requestUri">The Uri the request is sent to.</param>
+        /// <param name="jsonTypeInfo">Source generated JsonTypeInfo to control the behavior during deserialization.</param>
```

Comment:
```suggestion
        /// <param name="jsonTypeInfo">Source generated JSON type information to control the behavior during deserialization.</param>
```

Reply:
This has been taken from shipped docs, word-by-word:

https://docs.microsoft.com/en-us/dotnet/api/system.net.http.json.httpclientjsonextensions.getfromjsonasync?view=net-6.0#system-net-http-json-httpclientjsonextensions-getfromjsonasync(system-net-http-httpclient-system-string-system-type-system-text-json-serialization-jsonserializercontext-system-threading-cancellationtoken)

If we do make any changes here, they should be made across the board.